app/src/main/cpp/src/components/security_manager/test/ssl_certificate_handshake_test.cc

```diff
@@ -413,7 +413,7 @@ TEST_P(SSLHandshakeTest, CAVerification_ServerSide) {
   GTEST_TRACE(HandshakeProcedure_Success());
 }
 
-TEST_P(SSLHandshakeTest, CAVerification_ServerSide_NoCACertificate) {
+TEST_P(SSLHandshakeTest, DISABLED_CAVerification_ServerSide_NoCACertificate) {
```

Comment:
@LitvinenkoIra is this a part of changes from OpenSDL?

Reply:
Yes, this is a part of `https://github.com/smartdevicelink/sdl_core/pull/3582`